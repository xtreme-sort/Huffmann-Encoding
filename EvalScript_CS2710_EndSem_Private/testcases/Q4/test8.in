GenerateLengthLimitedHuffman
phqyubkukxzgdhiooyuymmflcixjpwnayehiaemmpxtuerorhcfbkguxkqeipkmzgvjgalhjedmuhhplpwubvwjlnjxqpmxqzjysrpgrxrwgioqeugforhytfjlxubaefrorelahbbulznjpihlyilsinklvjwppnvxsmouhsidftlavxdtvnpawbdeoxucgsdzsedrgilvsphhslwvyuibrdzwuquxjjmuhgnfuijetzdvrfjroadcapqxctciitptjwjychqckawxncixmugzdgsgkmzputctyivalizupdlqbozbmwiyyodytywkqqucrrxsokqrhopbztlqffjgvhktfsspwoyzgjhkcdoitsxzczpvrhjvuvfnknmpbrwgazxstcpqudjferbpbwcwxuyhimquekabxvafuhfmjgvjqobixrkngreenwsitmwqglpbgknabnn
6
Quit
