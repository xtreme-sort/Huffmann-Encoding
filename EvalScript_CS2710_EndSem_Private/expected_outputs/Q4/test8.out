h 0000
i 0001
j 0010
p 0011
u 0100
x 0101
a 01100
b 01101
c 01110
d 01111
e 10000
f 10001
g 10010
k 10011
l 10100
m 10101
n 10110
o 10111
q 11000
r 11001
s 11010
t 11011
v 11100
w 11101
y 11110
z 11111
cost: 2175
